apps/wallet/src/background/accounts/zk/utils.ts

```diff
@@ -0,0 +1,325 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
Most of the functionality here was ported from [here](https://github.com/MystenLabs/zklogin-circuits)